README.md

```diff
@@ -77,13 +80,13 @@ To be a NLP sandbox data hosting site, the site must be able to host 4 main tech
 
 ### SynapseWorkflowOrchestrator
 
-
+0. Obtain a Service Accout from the NLPSandbox Team 
 1. Clone the repository
     ```bash
     git clone https://github.com/Sage-Bionetworks/SynapseWorkflowOrchestrator.git
     cd SynapseWorkflowOrchestrator
     ```
-2. Add to the `docker-compose.yaml`.  The `ROUTE_URIS` will be different from the `Sage Bionetworks` site.
+2. Add the following section to the `docker-compose.yaml`ONLY if you run an ELK Logging server.  The `ROUTE_URIS` will be different from the `Sage Bionetworks` site.
```

Comment:
@gkowalski,  ELK is kind of a requirement, but there are certainly other tools that we can use to preserve the annotation tool service logs so that they can be returned to participants if necessary: https://docs.docker.com/config/containers/logging/configure/